src/Models/ArtistsModel.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require_once('src/Entities/Artist.php');
```

Comment:
Don't forget strict types! And some line breaks up here between require and the class